src/index.html

```diff
@@ -19,9 +19,381 @@
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            class="header__nav-item"
+            data-qa="nav-hover"
+          >
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Laptops&nbsp;&&nbsp;Computers
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div
+        class="catalog__card"
+        data-qa="card"
+      >
+        <img
+          class="catalog__card-image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac"
+        />
+
+        <h2 class="catalog__card-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="catalog__card-code">Product code: 195434</p>
+
+        <div class="catalog__card-rating">
+          <div class="catalog__card-stars">
+            <div class="catalog__card-stars-star"></div>
+            <div class="catalog__card-stars-star"></div>
+            <div class="catalog__card-stars-star"></div>
+            <div class="catalog__card-stars-star"></div>
+            <div class="catalog__card-stars-star"></div>
+          </div>
+          <span class="catalog__card-reviews">Reviews: 5</span>
+        </div>
+        <div class="catalog__card-price-container">
+          <span class="catalog__card-price-label">Price:</span>
+          <span class="catalog__card-price-amount">$2,199</span>
+        </div>
+
+        <a
+          href="#"
+          class="catalog__card-button"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
+      <div class="catalog__card">
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div class="catalog__card">
```